signup.js

```diff
@@ -0,0 +1,108 @@
+
+const userEmailField = document.querySelector('#email');
+const emailErrorMessage = document.querySelector('.email-field .error-message');
+const userNicknameField = document.querySelector('#nickname');
+const nicknameErrorMessage = document.querySelector('.nickname-field .error-message')
+const userPasswordField = document.querySelector('#password');
+const passwordErrorMessage = document.querySelector('.password-field .error-message');
+const userPasswordCheckField = document.querySelector('#passwordCheck');
+const passwordCheckErrorMessage = document.querySelector('.passwordCheck-field .error-message');
+const signupBotton = document.querySelector(".sign-up-button");
+
+const validEmail = /^[0-9a-zA-Z]([-_.]?[0-9a-zA-Z])*@[0-9a-zA-Z]([-_.]?[0-9a-zA-Z])*.(\.[a-zA-Z]{2,3})$/;
+
+document.getElementById('email').autocomplete = 'off';
+document.getElementById('nickname').autocomplete = 'off';
+document.getElementById('password').autocomplete = 'off';
+document.getElementById('passwordCheck').autocomplete = 'off';
+
+//버튼 유효성 검사
+let isEmailValid = false;
+let isNicknameValid = false;
+let isPasswordValid = false;
+let isPasswordCheckValid = false;
+
+function handleFormEmail(event) {
+    const input = userEmailField.value;
+    if ( input === '') {
+        emailErrorMessage.textContent = "이메일을 입력해주세요";
+        emailErrorMessage.style.color = 'red';
+        userEmailField.style.border = '1px solid red';
+    } else if (validEmail.test(input) ) {
+        emailErrorMessage.textContent = "올바른 이메일 형식입니다";
+        emailErrorMessage.style.color = '#3692FF';
+        userEmailField.style.border = '1px solid #3692FF';
+        isEmailValid = true;
+    } else {
+        emailErrorMessage.textContent = "잘못된 이메일 형식입니다";
+        emailErrorMessage.style.color = 'red';
+        userEmailField.style.border = '1px solid red';
+    }
+    handleSignupButton()
+}
+
+function handleFormPassword(e) {
```

Comment:
💊 제안
event 인자를 해당 함수내에서 사용하지 않으니 안 받으시는 것이 좋겠습니다~
```suggestion
function handleFormPassword() {
```